mapeditor/mainwindow.cpp

```diff
@@ -16,6 +16,9 @@
 #include <QFile>
 #include <QMessageBox>
 #include <QFileInfo>
+#include <qdialog.h>
+#include <qlistwidget.h>
+#include <qnamespace.h>
```

Comment:
please use the extension-less includes

Reply:
also not sure what qnamespace.h is for

Reply:
Fixed with [e3b520a](https://github.com/vcmi/vcmi/pull/4939/commits/e3b520a1f10a844488ba9e8250b3d70c3d866d52)